dcs/country.py

```diff
@@ -140,18 +140,14 @@ def next_callsign_category(self, category):
             self.current_callsign_category[category] = 0
         return self.callsign.get(category)[self.current_callsign_category[category]]
 
-    @property
-    def unused_onboard_numbers(self) -> Set[int]:
```

Comment:
any reason why this property was removed?

Reply:
I searched high and low for usages and couldn't find any. I can re-add of this is used by consumers.

Reply:
well pydcs is a library and so there will be a lot of methods/functions that are potentially not used within the library

Reply:
Reintroduced this property